homeassistant/components/renson/binary_sensor.py

```diff
@@ -0,0 +1,138 @@
+    RensonBinarySensorEntityDescription(
+        translation_key="frost_protection_active",
+        key="FROST_PROTECTION_FIELD",
+        field=FROST_PROTECTION_FIELD,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    RensonBinarySensorEntityDescription(
+        key="BREEZE_ENABLE_FIELD",
+        translation_key="breeze",
+        field=BREEZE_ENABLE_FIELD,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    RensonBinarySensorEntityDescription(
+        key="BREEZE_MET_FIELD",
+        translation_key="breeze_conditions_met",
+        field=BREEZE_MET_FIELD,
+    ),
+    RensonBinarySensorEntityDescription(
+        key="HUMIDITY_CONTROL_FIELD",
+        translation_key="humidity_control",
+        field=HUMIDITY_CONTROL_FIELD,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    RensonBinarySensorEntityDescription(
+        key="AIR_QUALITY_CONTROL_FIELD",
+        translation_key="air_quality_control",
+        field=AIR_QUALITY_CONTROL_FIELD,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    RensonBinarySensorEntityDescription(
+        key="CO2_CONTROL_FIELD",
+        translation_key="co2_control",
+        field=CO2_CONTROL_FIELD,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    RensonBinarySensorEntityDescription(
+        key="PREHEATER_FIELD",
+        translation_key="preheater",
+        field=PREHEATER_FIELD,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Call the Renson integration to setup."""
+
+    api: RensonVentilation = hass.data[DOMAIN][config_entry.entry_id].api
+    coordinator: RensonCoordinator = hass.data[DOMAIN][
+        config_entry.entry_id
+    ].coordinator
+
+    entities: list = []
+    for description in BINARY_SENSORS:
+        entities.append(RensonBinarySensor(description, api, coordinator))
+
+    async_add_entities(entities)
+
+
+class RensonBinarySensor(RensonEntity, BinarySensorEntity):
+    """Get sensor data from the Renson API and store it in the state of the class."""
+
+    _attr_has_entity_name = True
+    _attr_name = None
+
+    def __init__(
+        self,
+        description: RensonBinarySensorEntityDescription,
+        api: RensonVentilation,
+        coordinator: RensonCoordinator,
+    ) -> None:
+        """Initialize class."""
+        super().__init__(description.key, api, coordinator)
+
+        self.field = description.field
+        self.entity_description = description
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        all_data = self.coordinator.data
+
+        value = self.api.get_field_value(all_data, self.field.name)
+
+        self._attr_is_on = self.api.parse_value(value, DataType.BOOLEAN)
+
+        self.async_write_ha_state()
```

Comment:
```suggestion
```

Reply:
Without this line the binary sensors are not updated.

Reply:
Right. Because you don't have super.
You should have `super()._handle_coordinator_update()` instead so it calls in the base class update.
If the base coordinator get's updated somewhere down the line then, this integration will continue to work.